DateTimeMeter.c

```diff
@@ -0,0 +1,41 @@
+/*
+htop - DateTimeMeter.c
+(C) 2004-2011 Hisham H. Muhammad
+Released under the GNU GPL, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "DateTimeMeter.h"
+
+#include "CRT.h"
+
+#include <time.h>
+
+
+int DateTimeMeter_attributes[] = {
+   DATE
+};
+
+static void DateTimeMeter_updateValues(Meter* this, char* buffer, int size) {
+   time_t t = time(NULL);
+   struct tm result;
+   struct tm *lt = localtime_r(&t, &result);
+   this->values[0] = lt->tm_year * 365 * 24 * 60 + lt->tm_yday * 24 * 60 +
+                      lt->tm_hour * 60 + lt->tm_min;
```

Comment:
This looks odd regarding leap years …

Reply:
Yes this is just a rough estimate. The bigger issue is however: what should the numeric value be and what should the maximum be. They are mostly irrelevant since people will normally only use this meter in text mode not as graphs – but they have to be specified. Some options I came up with:
- Currently, the maximum is 100 (copied from UptimeMeter) and the value is roughly seconds after 1900-01-01, meaning that the meter it always fully filled.
- Set maximum to some arbitrary timepoint like 2050, 2100 or 3000. → Partly filled, seemingly stationary
- Set maximum to LONG_MAX and value to 0 → Not filled
- Set the maximum to the end of the current year, value to minutes (or days) since new year → Partly filled, starts from start on new year. [I prefer this, since it's the only one that could be of any use]

Handling leap years properly is imho most likely not worth the effort. (But possible by `(year % 4 == 0) && (year % 100!= 0)) || (year%400 == 0)`)

Reply:
I now implemented the later – with leap years.